src/integrations/ga/GaToSplit.js

```diff
@@ -1,7 +1,257 @@
-// @TODO implement real module
-// eslint-disable-next-line no-unused-vars
-function GaToSplitFactory(options, storage, coreSettings) {
-  return {};
+import { isString, isNumber, uniqO } from '../../utils/lang';
+import logFactory from '../../utils/logger';
+import {
+  validateEvent,
+  validateEventValue,
+  validateEventProperties,
+  validateKey,
+  validateTrafficType,
+} from '../../utils/inputValidation';
+const log = logFactory('splitio-ga-to-split');
+
+/**
+ * Provides a plugin to use with analytics.js, accounting for the possibility
+ * that the global command queue has been renamed or not yet defined.
+ * @param {string} pluginName The plugin name identifier.
+ * @param {Function} pluginConstructor The plugin constructor function.
+ */
+function providePlugin(pluginName, pluginConstructor) {
+  const gaAlias = window.GoogleAnalyticsObject || 'ga';
+  window[gaAlias] = window[gaAlias] || function (...args) {
+    (window[gaAlias].q = window[gaAlias].q || []).push(args);
+  };
+
+  // provides the plugin for use with analytics.js.
+  window[gaAlias]('provide', pluginName, pluginConstructor);
+
+  // @TODO should we registers the plugin on the global gaplugins object?
```

Comment:
I would think so, what are the implications?

Reply:
Removed. autotrack uses it for testing purposes.